scripts/core/exp-command-buffer.yml

```diff
@@ -163,7 +314,11 @@ params:
       desc: "[in][optional] A list of sync points that this command depends on."
     - type: "$x_exp_command_buffer_sync_point_t*"
       name: pSyncPoint
-      desc: "[out][optional] sync point associated with this command"
+      desc: "[out][optional] Sync point associated with this command."
+    - type: "$x_exp_command_buffer_command_handle_t*"
+      name: phCommand
+      desc: "[out][optional] Handle to this command."
```

Comment:
Is there a danger here that users may think they have ownership over this handle? Since we're differing from the normal Create/Retain/Release methods of retrieving other handle types.

Perhaps a regular pointer type like with the sync point would prevent any confusion instead of calling it a handle.

Reply:
That's a fair point, calling a type a handle that isn't being created/retained/release could be confusing. Will remove the handler terminology from this type and it's usage

Reply:
After discussion I've added retain/release entry-points to command-handles. This makes them analogous to events in that they are created by being returned from command adding entry-points and reference counted. The relationship between command handle objects and command-buffer objects is the same as kernel/programs in that creating a command handle increments the reference count of the command-buffer.

I've updated the spec with the above, and done a quick impl but for the CUDA adapter but need to write a CTS tests or two to exercise this.